composer.json

```diff
@@ -17,6 +17,7 @@
     "autoload": {
         "psr-4": {
             "React\\EventLoop\\": "src"
-        }
+        },
+        "files": ["src/functions.php"]
```

Comment:
See reactphp/promise#23 and reactphp/promise#25. Might be fixed in the meantime by composer/composer#4186.